modules/rosa-node-drain-grace-period.adoc

```diff
@@ -0,0 +1,82 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa_nodes/rosa-managing-worker-nodes.adoc
+//this module applies to ROSA HCP only
+
+:_mod-docs-content-type: PROCEDURE
+[id="rosa-node-drain-grace-period_{context}"]
+= Configuring node drain grace periods in {hcp-title} clusters
+
+You can configure the node drain grace period for machine pools in your cluster. The node drain grace period for a machine pool is how long the cluster respects the Pod Disruption Budget protected workloads when upgrading or replacing the machine pool. After this grace period, all remaining workloads are forcibly evicted. The value range for the node drain grace period is from 0 to 1 week. With the default value 0, or empty value, the machine pool drains without any time limitation until complete.
+
+
+.Prerequisites
+
+* You installed and configured the latest {product-title} (ROSA) CLI, `rosa`, on your workstation.
+* You created a {hcp-title-first} cluster.
+* You have an existing machine pool.
+
+.Procedure
+
+. List all of the machine pools in the cluster:
```

Comment:
Per our [docs guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#procedure-step-that-introduces-a-command), steps that introduce a command need some language that explicitly mention a command needs to be run:
```suggestion
. List all of the machine pools in the cluster by running the following command:
```